Header header
string[] class_name_of_the_box
string[] x_positions_of_all_class
string[] y_positions_of_all_class
string[] z_positions_of_all_class